ui/components/multichain/create-btc-account/create-btc-account.tsx

```diff
@@ -14,18 +14,28 @@ type CreateBtcAccountOptions = {
    * Callback called once the account has been created
    */
   onActionComplete: (completed: boolean) => Promise<void>;
+  /**
+   * CAIP-2 chain ID
+   */
+  network: MultichainNetworks;
+  /**
+   * Default account name
+   */
+  defaultAccountName?: string;
```

Comment:
Shouldn't the account name be mandatory? I think it can be confusing if we use the default value for a testnet account.

Reply:
True! Lemme change this to a required prop